phpcs.xml

```diff
@@ -27,18 +27,9 @@
         </properties>
     </rule>
 
-    <!-- exclude for now -->
-    <rule ref="Spryker.ControlStructures.NoInlineAssignment">
-        <severity>0</severity>
-    </rule>
-    <rule ref="SlevomatCodingStandard.ControlStructures.AssignmentInCondition.AssignmentInCondition">
-        <severity>0</severity>
-    </rule>
-    <rule ref="Generic.PHP.NoSilencedErrors">
-        <severity>0</severity>
-    </rule>
-    <rule ref="Squiz.PHP.Eval">
-        <severity>0</severity>
-    </rule>
+    <rule ref="Spryker.ControlStructures.NoInlineAssignment"/>
```

Comment:
Why are you adding them again?
They are already included by default, thats why they are silenced :)